libsentrykube/service.py

```diff
@@ -1,17 +1,49 @@
 from pathlib import Path
-from typing import List, Mapping, Any
+from typing import List, Mapping, Any, Union
 
 import click
 import yaml
 
 from collections import OrderedDict
 from libsentrykube.config import Config
 from libsentrykube.customer import load_customer_data
-from libsentrykube.utils import workspace_root
+from libsentrykube.utils import workspace_root, deep_merge_dict
 
 _services = OrderedDict()
 
 
+def assert_single_cluster_for_customer(
+    service_name: str, customer_name: str, cluster_name: str, external: bool = False
+) -> None:
+    """
+    Make sure that a cluster can only exist once for a service.
+    Checks the `region_overrides` and every folder directly in `region_overrides`
+    """
+    if external:
+        service_regions_path = workspace_root() / service_name
+    else:
+        service_regions_path = get_service_path(service_name)
+
+    paths: List[Path] = []
+    paths.extend(
+        service_regions_path.glob(
+            f"region_overrides/{customer_name}/{cluster_name}.yaml"
+        )
+    )
+    paths.extend(
+        service_regions_path.glob(
+            f"region_overrides/*/{customer_name}/{cluster_name}.yaml"
+        )
+    )
+
+    if len(paths) > 1:
+        click.echo(f"Expected a single cluster file for customer but got {len(paths)}")
+        click.echo("Found files in:")
```

Comment:
Please do not print on the console at this level. 
`libsentrykube` is a library and it is independent on how the result is presented.
The cli scripts are outside this package.

Please raise exceptions instead.